app/livechat/server/api/lib/livechat.js

```diff
@@ -107,10 +110,24 @@ export function normalizeHttpHeaderData(headers = {}) {
 	const httpHeaders = Object.assign({}, headers);
 	return { httpHeaders };
 }
-export async function settings() {
+export async function settings(businessUnitIds = '') {
```

Comment:
```suggestion
export async function settings({ businessUnit = '' }) {
```

Array is not expected, just one single business unit (Id / Name) will be passed as parameter.